libraries/webInterface/gui/index.js

```diff
@@ -85,6 +85,43 @@ realityServer.initialize = function () {
     this.initializeHelp();
 };
 
+
+realityServer.activeUiElement = function() {
```

Comment:
Most web APIs would capitalize this activeUIElement

Reply:
function does not exist anymore